### DISCUSSION TITLE
RustDesk Silent Install (own server or public server)- windows 7/8

### DISCUSSION THREAD

**Discussion by abdullah-erturk** [rustdesk/rustdesk#10102] [Show and tell]
The latest 64-bit version of RustDesk now gives an error during installation due to the lack of Flutter support on Windows 8.1 and lower versions.
For detailed information: https://github.com/rustdesk/rustdesk/discussions/763

However, there are still many computers using Windows 7.
For this reason, I updated my Rustdesk silent installation script file that I prepared a long time ago.

Now, 32-bit Rustdesk will be installed on Windows 8.1 and earlier versions, and 64-bit Rustdesk will be installed on Windows 10 and later versions.

With this script file, you will be able to install the latest version of Rustdesk without any problems. Also, if you have your own Rustdesk server, if you add your server information as a variable in the script file, the Rustdesk software will run directly on your server.

https://github.com/abdullah-erturk/RustDesk-Silent-Installation/tree/main

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/10085#issuecomment-2506485955

**Reply by abdullah-erturk** [reply]
I updated the script to fix the issue I mentioned in #10085 (comment).

**Reply by rustdesk** [reply]
We are attempting to modify Flutter engine to make it run on win7, if we are successful, will update here.

**Reply by rustdesk** [reply]
https://github.com/rustdesk/rustdesk/releases/download/nightly/rustdesk-1.3.4-x86_64.exe work for you?

**Reply by abdullah-erturk** [reply]
Tested on Windows 7 x64.

It works perfectly, Thank you very much.
